src/components/ParticipantList.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { DataGrid } from "@mui/x-data-grid";
+import Container from "@mui/material/Container";
+import { useSelector } from "react-redux";
+
+const ParticipantList = () => {
+  const { participantList } = useSelector((state) => state);
+  const { departments } = useSelector((state) => state);
+
+  const columns = [
+    {
+      field: "name",
+      headerName: "Name",
+      flex: 1,
+    },
+    {
+      field: "score",
+      headerName: "Fika Score",
+      flex: 1,
+    },
+  ];
+
+  let rows = departments[participantList.id - 1].participants.map((participant) => {
```

Comment:
If `participantList` is a list, how can you call `.id` on it? I'm not sure what is going on here

Reply:
Maybe not the best naming, it'll be easier to show you than type it out